platform/broadcom/one-pde-image.mk

```diff
@@ -0,0 +1,81 @@
+# sonic broadcom one image installer
+
+SONIC_ONE_PDE_IMAGE = sonic-broadcom-pde.bin
```

Comment:
it seems it is not necessary to define a entire new sonic pde image for this purpose.
we can use existing sonic image and introduce a INCLUDE_SONIC_CORE = n to disable the all the core components, then we can add INCLUDE_PDE that only include PDE docker.

This could be a cleaner design.

Reply:
Removed